domains/measurements/Measurements.API/API/Program.cs

```diff
@@ -1,69 +1,33 @@
-using System.Runtime.CompilerServices;
-using System.Text.Json;
-using System.Text.Json.Serialization;
-using API.Options;
-using API.Services;
-using FluentValidation;
-using FluentValidation.AspNetCore;
-using MicroElements.Swashbuckle.FluentValidation.AspNetCore;
-using Microsoft.Extensions.Diagnostics.HealthChecks;
-using Microsoft.Extensions.Options;
+using API.Extensions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
 using Serilog;
-using Serilog.Enrichers.Span;
-using Serilog.Formatting.Json;
+using System;
 
-[assembly: InternalsVisibleTo("Tests")]
+var environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+var configuration = new ConfigurationBuilder()
+    .AddJsonFile("appsettings.json", optional: false)
```

Comment:
Why this?

Reply:
To use correct appsettings.json depending on the environment